### ISSUE TITLE
Issue Reproducing 'starry' Style Transfer on 'garden' Scene: High Loss Value Observed

### ISSUE THREAD

**Issue by OstarryskyO** [JianlingWANG2021/SGSST#2]
I have been exploring your open-source project, "SGSST: Scaling Gaussian Splatting Style Transfer," and appreciate the innovative approach it offers. However, I encountered an issue while attempting to reproduce the stylization results. Specifically, when applying the 'starry' style to the 'garden' scene, the output differs from the expected results. During the stylization process, I observed that the loss value was 5,856,032,888. Below is the command I used for stylization:
python stylize.py \
--source_path /data/gaussian-splatting-dev/datasets/garden \
--model_path output/model_garden_103 \
--style_img starry.jpg \
--iterations 50000 \
--resolution 1 \
--port 4751
I would be grateful for any guidance or suggestions you could provide to help resolve this discrepancy.

**Comment by bgalerne** [comment]
What is important is to use style images with size larger than the content images from the 3DGS. 
For starry night we used a downscaled version with 6k pixels of the original 30k image from here:
https://en.wikipedia.org/wiki/File:Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg
Hope this helps.

**Comment by OstarryskyO** [comment]
I'm trying to apply a larger style image than the content images, similar to the one shown below:
![Image](https://github.com/user-attachments/assets/3f8213b2-eb62-4695-8e29-d0261b1a06f9)
However, when I run the following command, the results are not as expected:
--source_path "/data/datasets/garden"
--model_path "output/model_garden_starry"
--style_img "style/starry.jpg"
--iterations 50000
--resolution 1
--port 4751
The "starry.jpg" style image(original size 30k) is downloaded from the URL you provided. Despite using a larger style image, the output stylized image looks like this:
![Image](https://github.com/user-attachments/assets/630d5ec4-2cb0-4673-9be5-50c00370d306)
Could you please help me figure out why the stylization isn't behaving as expected? Any suggestions would be greatly appreciated!
Thanks!

**Comment by bgalerne** [comment]
OK. 
Have you used a "--start_checkpoint <path of the 3DGS input model>" to start the stylization of a pre-trained realistic 3DGS ?
Also regarding sizes, the input garden images should be 5187x3361, so the reported sizes are too small.

**Comment by OstarryskyO** [comment]
I added --start_checkpoint, and it worked! Thanks for the help!
